examples/nas/enas/utils.py

```diff
@@ -0,0 +1,27 @@
+import torch
```

Comment:
You don't mention the torch is a dependency in this example?

Reply:
Everything is dependent on PyTorch.